solutions/usaco-965.mdx

```diff
@@ -88,4 +91,51 @@ int main() {
 		cout << cows[t] << '\n';
 	}
 }
-```
\ No newline at end of file
+```
+</CPPSection>
+
+<PySection>
+## Brute Force with Itertools
+
+<Warning>
+Due to the brute force nature of this solution, it is very slow and is not the optimal solution.
+</Warning>
+We can easily iterate through every possible permutation of cows in lexographic order using Itertools. Then we can check for the first permutation that meets the adjacency constraints and this will be our answer. 
+
+**Time Complexity:** $\mathcal{O}(NC*C!)$, where $C$ represents the number of cows 
```

Comment:
```suggestion
**Time Complexity:** $\mathcal{O}(NC \cdot C!)$, where $C$ represents the number of cows 
```
also, shouldn't this just be N * C! ?

Reply:
I use .index tho so isnt it multiplied by C?